ocaml/xapi/repository.ml

```diff
@@ -147,26 +147,39 @@ let get_proxy_params ~__context repo_name =
   | _ ->
       ("", "", "")
 
-let sync ~__context ~self ~token ~token_id =
+let ext_host_verified_rpc ~__context ~cert host_address xml =
+  try Helpers.make_external_host_verified_rpc ~__context host_address cert xml
+  with Xmlrpc_client.Connection_reset ->
```

Comment:
Now when the certificate is invalid, the expected `ssl_verify_error` is not raised, instead this `Xmlrpc_client.Connection_reset` is caught here. I think this problem can be resolved with another ticket later.